legacy/src/Charts.js

```diff
@@ -22,8 +22,10 @@ import {
   ChartEmits
 } from '../../src/utils'
 
+import { ComponentStore } from './store.ts'
+
 export function generateChart(chartId, chartType, chartController) {
-  let _chart = null
+  let _store = null
```

Comment:
it looks like "ref", so then maybe just

```
let _chartRef = { current: null }
...
_chartRef.current
```

?

Reply:
@dangreen, I checked, and your solution doesn't fix this bug.

Reply:
ok, here just `let _chartRef = null`

`_store = new ComponentStore()` -> `_chartRef = { current: null }`
`_store.setComponentChart(...)` -> `_chartRef.current = ...`
etc

Reply:
It works.  I used it.